chainerx/_docs/routines.py

```diff
@@ -1354,7 +1354,7 @@ def _docs_math():
     x2 (~chainerx.ndarray): Input array.
 
 Returns:
-    :class:`~chainerx.ndarray`: Returned array: :math:`y = \\arctan (x1/x2)`.
+    :class:`~chainerx.ndarray`: Element-wise arc tangent of x1/x2.
```

Comment:
It seems still wrong because `arctan` is different from `arctan2`.

Reply:
Oh, how about 
`Array of angles in radians, in the range [-pi, pi].`
Blatantly copied from numpy docs. :stuck_out_tongue:

Reply:
How about explaining like as follows?
```
Returns an array represents theta in the range [-pi, pi],
such that x1 = r cos(theta), x2 = r sin(theta) for some r > 0. 
```

Reply:
Sure. That makes sense. Will update to that.